docs/source/gbd2017_models/causes/smoking_affected_causes/index.rst

```diff
@@ -0,0 +1,459 @@
+     - 
+     - 
+   * - haemorrhagic stroke (Intracerebral hemorrhage)
+     - 496
+     - 2
+     - 
+     - 
+     - 
+   * - subarachnoid haemorrhage
+     - 497
+     - 2
+     - 
+     - 
+     - 
+   * - atrial fibrillation and flutter
+     - 500
+     - 11
+     - 
+     - 
+     - Age group start after lung cancer
+   * - aortic aneurysm
+     - 501
+     - 8
+     - 
+     - 
+     - 
+   * - peripheral arterial disease
+     - 502
+     - 13
+     - 
+     - 
+     - Age group start after lung cancer
+   * - chronic obstructive pulmonary disease
+     - 509
+     - 4
+     - 
+     - 
+     - 
+   * - other chronic respiratory diseases
+     - 520
+     - 2
+     - 
+     - 
+     - 
+   * - asthma
+     - 515
+     - 5
+     - 
+     - 
+     - 
+   * - peptic ulcer disease
+     - 527
+     - 5
+     - 
+     - 
+     - 
+   * - gallbladder and biliary tract diseases
+     - 534
+     - 5
+     - 
+     - 
+     - 
+   * - Alzheimer disease and other dementias
+     - 543
+     - 13
+     - 
+     - 
+     - Age group start after lung cancer
+   * - Parkinson disease (protective)
+     - 544
+     - 9
+     - 
+     - 
+     - Age group start after lung cancer
+   * - multiple sclerosis
+     - 546
+     - 9
+     - 
+     - 
+     - Age group start after lung cancer
+   * - type‐II diabetes
+     - 976
+     - 8
+     - 
+     - 
+     - 
+   * - rheumatoid arthritis
+     - 627
+     - 6
+     - 
+     - 
+     - 
+   * - low back pain
+     - 630
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - cataracts
+     - 671
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - macular degeneration
+     - 672
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+   * - Fracture
+     - N/A
+     - N/A
+     - YLD only
+     - 
+     - Exclude due to nonfatal
+
+.. note::
+
+	The starting age group IDs were extracted according to the YLL start, not the YLD start.
+
+	The :ref:`Vivarium CSU Lung Cancer Screening  Model  <2017_concept_model_vivarium_swissre_lungcancer>` only models simulants 15 years (age group ID 8) and older, so notes were made for causes with age group starts older than 15 years only.
+
+Restrictions
+++++++++++++
+
+See the table above.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+The Vivarium modeling strategy outlined in this document will be a **mortality only** model that combines several causes within GBD. This cause is intended to be modeled as a outcome pair with the smoking risk factor.
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+.. todo::
+
+	List assumptions and limitations
+
+Cause Model Diagram
++++++++++++++++++++
+
+There is no cause model diagram for this cause model because it is a mortality-only model. Mortality using should be modeled as follows:
+
+.. math::
+
+	mr_i = ACMR - CSMR_\text{SACs} + CSMR_\text{SACs} * (1 - PAF) * RR_i - \sum_{c}^{n} CSMR_c + \sum_{c}^{n} EMR(i)_c
```

Comment:
I'm wondering if we build this equation up so its clear how this is derived (kind of like how we did for the LBWSG documentation)? My understanding is as thus (correct me if I'm wrong!):

unmod: unmodelled causes
unaff: unaffected causes
aff: affected (here it is the SACs)
mod: modelled (here it is lung cancer)

mr_i = ACMR 
        = sum_CSMR_unmod_unaff_causes + sum_CSMR_smoking_aff_causes + sum_CSMR_mod_causes (eg. lung cancer)

sum_CSMR_unmod_unaff_causes =  ACMR - sum_CSMR_smoking_aff_causes - sum_CSMR_mod_causes (eg. lung cancer)

sum_CSMR_smoking_aff_causes = CSMR*(1-PAF)*RR (does this need a summation or is it pooled somehow? this is lung cancer deleted right?)

sum_CSMR_mod_causes (eg. lung cancer) = sum_EMR(state)

(should there be a summation sign for CSMR_sacs ?)